benefits/core/session.py

```diff
@@ -23,8 +23,8 @@
 _DEBUG = "debug"
 _DID = "did"
 _ELIGIBLE = "eligibility"
-_ENROLLMENT_TOKEN = "enrollment_token"
-_ENROLLMENT_TOKEN_EXP = "enrollment_token_expiry"
+_LITTLEPAY_ENROLLMENT_TOKEN = "littlepay_enrollment_token"
+_LITTLEPAY_ENROLLMENT_TOKEN_EXP = "littlepay_enrollment_token_expiry"
```

Comment:
Just a note here:

In a later PR for a different ticket (#2843), we will need to introduce Switchio session variables as well to hold the registration ID that the user is using.

I think eventually we will add new Django apps for `littlepay` and `switchio`, and those apps would have a `session` module that maintains these variables rather than them being in `core.session`.

Reply:
Yeah I had the same thought, and I'd like to use this PR to start on that. We already have #2800 and I feel like this is just adding more and more technical debt to our session handling. I'd rather we avoid this kind of thing going forward, knowing that we want to clean all this up.

There is no reason for any of this to be in the `core` module right? No other parts of the code need to know about this except for `enrollment`?

Reply:
(I'm not necessarily saying to implement a class-based session here, but let's at least keep this new stuff separate and where it is needed, vs. introducing new global/`core` level stuff when it isn't needed there).

Reply:
So are you thinking we should go ahead and introduce `littlepay` and `switchio` apps to hold their specific things (e.g. models, templates, session, etc.)? Or are you thinking of a smaller scope, like adding a `littlepay_session.py` and `switchio_session.py`? 

(You probably don't mean the latter since you said 
> There is no reason for any of this to be in the `core` module right? 

)

Reply:
I think we should be doing the thing we think is the best long-term approach, not the thing we think is the way to get it done now and hopefully we'll have time to come back and clean it up. We are implementing new code/functionality now. Let's do it the cleanest way possible. Let's keep in mind all the stuff we've been talking about: smaller, more focused apps. Less global/shared `core` functionality. 

Not to mention, having all this code new in its own modules/apps/whatever, just imported and called in `enrollment`, would make reviews _so much easier_. I think testing would probably be simpler as well.

Reply:
:point_up: that is a general comment for this entire PR, not just this specific session file.

Reply:
Yeah I think we just don't have as much luxury of time anymore with all the other things going on, there's less chance we have to come back and clean up things.

Reply:
That makes sense. Thanks for this feedback, and I'll look into how to organize things so that the code is cleaner.